models/discounted-price/src/fields-config.ts

```diff
@@ -0,0 +1,30 @@
+import { Reference, Price } from '@commercetools-test-data/commons';
+import { fake, TModelFieldsConfig } from '@commercetools-test-data/core';
+import type { TDiscountedPriceGraphql, TDiscountedPriceRest } from './types';
+
+/**
+ * value - Reference to Money
+ * discount - Reference to a ProductDiscount Reference Object {typeId, id}
+ */
+const commonFieldsConfig = {
+  value: fake(() => Price.random()),
+  discount: fake(() => Reference.presets.productDiscountReference()),
```

Comment:
We need to bear in mind the shape of this property is different between REST and GraphQL so this can't be a shared one.